master/internal/experiment/bulk_action.go

```diff
@@ -44,6 +44,28 @@ type deleteExperimentOKResult struct {
 // ExperimentsAddr is the address to direct experiment actions.
 var ExperimentsAddr = actor.Addr("experiments")
 
+// terminalExperimentStates where an experiment can be deleted, archived, or unarchived.
+var terminalExperimentStates = func() []string {
+	states := make([]string, len(model.TerminalStates))
+	idx := 0
+	for state := range model.TerminalStates {
+		states[idx] = string(state)
+		idx++
+	}
+	return states
+}()
```

Comment:
can we just add some sort of states to string?

also remember to use `make([]T, 0, len(other))` and use append.